library/psa_crypto.c

```diff
@@ -3530,7 +3530,7 @@ static psa_status_t psa_rsa_verify( mbedtls_rsa_context *rsa,
 }
 #endif /* MBEDTLS_RSA_C */
 
-#if defined(MBEDTLS_ECDSA_C)
+#if defined(PSA_WANT_ALG_ECDSA)
```

Comment:
Shouldn't this be MBEDTLS_PSA_BUILTIN_ALG_ECDSA? It seems to me that this is the software implementation that is needed only if MBEDTLS_PSA_BUILTIN_ALG_ECDSA is defined.

Reply:
This might be true. The support code for actually implement MBEDTLS_PSA_ACCEL_ALG_ECDSA is not complete so I was not sure which portions would change to use BUILTIN vs just the WANT_ALG definition. I could update this one, but there are others also in this file that are also in question then. I thought that once the accelerator was implemented it would then be determined which pieces need the BUILTIN and which areas did not.